### ISSUE TITLE
Base mkdocs-material-insiders doesn't work

### ISSUE THREAD

**Issue by ianderrington** [squidfunk/mkdocs-material#6573]
### Context

I have used mkdocs-material for a while. I have some older venvs that have various version of it working and recently upgraded to insiders. I'm trying to build it, but found many build issues: blog didn't work, meta didn't work, etc... So I Decided to build a new venv, using it directly. 

### Bug description

Documentation does not build following the build instructions [here](https://squidfunk.github.io/mkdocs-material/insiders/getting-started/)

### Related links

- [Reporting a bug](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/)
-here](https://squidfunk.github.io/mkdocs-material/insiders/getting-started/)

### Reproduction

n/a Will not run 

### Steps to reproduce

```bash
conda create --name mkdocs-material-insiders  python=3.10 
conda activate mkdocs-material-insiders
git clone git@github.com:squidfunk/mkdocs-material-insiders.git mkdocs-material
pip install -e mkdocs-material --force-reinstall 
cd mkdocs-material
mkdocs build; mkdocs serve --livereload
```

```bash 
(mkdocs-material-insiders) ianderrington@ians-mbp mkdocs-material % mkdocs build; mkdocs serve --livereload
ERROR   -  Config value 'plugins': The "rss" plugin is not installed

Aborted with 1 configuration errors!
ERROR   -  Config value 'plugins': The "rss" plugin is not installed

Aborted with 1 configuration errors!
```

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/).
- [X] I have attached links to [the documentation](https://squidfunk.github.io/mkdocs-material/), and possibly related [issues](https://github.com/squidfunk/mkdocs-material/issues) and [discussions](https://github.com/squidfunk/mkdocs-material/discussions).
- [X] I assure that I have [removed all customizations](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/#remove-customizations) before submitting this bug report.
- [X] I have attached a __.zip file__ with a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/).

**Comment by alexvoss** [comment]
The RSS plugin is not part of Material for MkDocs and needs to be installed separately.

**Comment by ianderrington** [comment]
Oh, I thought doing the pip build would have included that, but I guess it isn't in the requirements...

**Comment by alexvoss** [comment]
Hi, mkdocs-rss-plugin is one of many plugins as you can see from the [MkDocs catalogue](https://github.com/mkdocs/catalog). We could not possibly include them all as dependencies. I recommend that you write your own `requirements.txt` for your projects in order to install what is needed and to document it.

**Comment by squidfunk** [comment]
That's correct. We try to keep our [`requirements.txt`](https://github.com/squidfunk/mkdocs-material/blob/master/requirements.txt) as small as possible, so installation and download are fast. All other dependencies must be explicitly installed. Note that we have some extras defined, the `recommended` of which contains the RSS plugin, but they are still subject to change which is why they are not yet documented:

https://github.com/squidfunk/mkdocs-material/blob/246266fafad2c2fdc78022c14f7f4b2df91167f3/pyproject.toml#L55-L68

Thus, if you're only missing the RSS plugin, you could do:

```
pip install "mkdocs-material[recommended]"
```

**Comment by ianderrington** [comment]
Thanks @squidfunk ! So elementary ... sorry to have taken your time!
